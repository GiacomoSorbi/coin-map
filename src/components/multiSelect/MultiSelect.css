.input-field {
  color: green;
  text-align: center;
  background-color: bisque;
  font-weight: normal;
  font-size: calc(6px + 1.5vmin);
  padding: 1vmin;
  border-radius: 2vmin;
  border: 1px gold solid;
  margin: 1vmin;
}
