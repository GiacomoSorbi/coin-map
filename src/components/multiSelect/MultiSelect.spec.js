import React from 'react';

import MultiSelect from './index';
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
  const fn = jest.fn();
  const renderer = new ShallowRenderer();
  const sampleRender1 = renderer.render(
    <MultiSelect
      height="400"
      onChange={fn}
      options={[
        {
          id: 0,
          label: 'pippi'
        },
        {
          id: 1,
          label: 'pappa'
        },
        {
          id: 2,
          label: 'food'
        },
        {
          id: 3,
          label: 'cat'
        },
        {
          id: 4,
          label: 'nanna'
        }
      ]}
    />
  );
  expect(sampleRender1).toMatchSnapshot();
});
