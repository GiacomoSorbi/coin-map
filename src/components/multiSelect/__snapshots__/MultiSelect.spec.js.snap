// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<n
  height={400}
  items={
    Array [
      Object {
        "id": 0,
        "label": "pippi",
      },
      Object {
        "id": 1,
        "label": "pappa",
      },
      Object {
        "id": 2,
        "label": "food",
      },
      Object {
        "id": 3,
        "label": "cat",
      },
      Object {
        "id": 4,
        "label": "nanna",
      },
    ]
  }
  onChange={[Function]}
  responsiveHeight="400px"
  selectedItems={Array []}
/>
`;
